### PR TITLE
[Enchancement](function) nullable inline refactor of min_max_by/bitmap && add register_functio…

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes

1. nullable inline refactor of min_max_by/bitmap/group_concat/histogram/topn
2. add register_function_both method
3. add datetimev2 type creator of min_max_by
4. remove uint16/32/64 in FOR_INTEGER_TYPES

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/vec/aggregate_functions/aggregate_function_bitmap.cpp [hunk 1]
```diff
@@ -18,50 +18,45 @@
 #include "vec/aggregate_functions/aggregate_function_bitmap.h"
 
 #include "vec/aggregate_functions/aggregate_function_simple_factory.h"
+#include "vec/aggregate_functions/helpers.h"
 
 namespace doris::vectorized {
 
 template <bool nullable, template <bool, typename> class AggregateFunctionTemplate>
 static IAggregateFunction* createWithIntDataType(const DataTypes& argument_type) {
```
Comment (HappenLee) [inline be/src/vec/aggregate_functions/aggregate_function_bitmap.cpp:26]: create_with_int_data_type

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**